src/sql/data_misc.sql

```diff
@@ -3138,7 +3138,7 @@ INSERT INTO `geodesic_site_settings` (`setting`, `value`) VALUES
 --
 
 INSERT INTO `geodesic_userdata` (`id`, `username`, `email`, `email2`, `newsletter`, `level`, `company_name`, `business_type`, `firstname`, `lastname`, `address`, `address_2`, `zip`, `city`, `state`, `country`, `phone`, `phone2`, `fax`, `url`, `date_joined`, `communication_type`, `feedback_score`, `feedback_count`, `feedback_positive_count`, `rate_sum`, `rate_num`, `optional_field_1`, `optional_field_2`, `optional_field_3`, `optional_field_4`, `optional_field_5`, `optional_field_6`, `optional_field_7`, `optional_field_8`, `optional_field_9`, `optional_field_10`, `affiliate_html`, `filter_id`, `expose_email`, `expose_company_name`, `expose_firstname`, `expose_lastname`, `expose_address`, `expose_city`, `expose_state`, `expose_country`, `expose_zip`, `expose_phone`, `expose_phone2`, `expose_fax`, `expose_url`, `expose_optional_1`, `expose_optional_2`, `expose_optional_3`, `expose_optional_4`, `expose_optional_5`, `expose_optional_6`, `expose_optional_7`, `expose_optional_8`, `expose_optional_9`, `expose_optional_10`, `account_balance`, `date_balance_negative`, `balance_freeze`, `feedback_icon`, `storefront_header`, `storefront_template_id`, `storefront_welcome_message`, `storefront_on_hold`, `storefront_home_link`, `storefront_traffic_processed_at`, `last_login_time`, `last_login_ip`, `verified`, `storefront_trials_used`) VALUES
-(1, 'admin', 'geoproducts@geodesicsolutions.com', '', 0, 1, 'Geodesic Solutions', 0, 'admin', 'admin', 'address', '', 'zipcode', 'city', 'state', 'country', 'phone', '', 'fax', '', 0, 3, 0, 0, 0, 0, 0, '', '', '', '', '', '', '', '', '', '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0.0000, 0, 0, '', '', 0, '', 1, '', 1164053097, '2006-11-21 17:53:07', '192.168.1.124', 'no', '');
+(1, 'admin', 'fake@geodesicsolutions.org', '', 0, 1, 'Geodesic Solutions', 0, 'admin', 'admin', 'address', '', 'zipcode', 'city', 'state', 'country', 'phone', '', 'fax', '', 0, 3, 0, 0, 0, 0, 0, '', '', '', '', '', '', '', '', '', '', '', 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0.0000, 0, 0, '', '', 0, '', 1, '', 1164053097, '2006-11-21 17:53:07', '192.168.1.124', 'no', '');
 
```

Comment:
Same here, if the URL default serves no real purpose, use `example.com`.  So in this case since it is for the default admin email, use `admin@example.com`.

Reply:
I seem to recall that there are also phony links in there that should also use example.com

www.MyListingsSite.com